client/src/features/login/LogIn.tsx

```diff
@@ -67,21 +81,20 @@ const LogIn = () => {
             </label>
             <div className="relative">
               <input
+                id="password"
                 type={isVisible ? 'text' : 'password'}
-                className="h-10 mb-12 w-full px-8"
-                onChange={(e) =>
-                  setInputs({ ...inputs, password: e.target.value })
-                }
+                className="h-10 mb-12 w-full px-8 rounded"
```

Comment:
maybe it's better to have the same font size as email input.

Reply:
Thank you, I added text-xl for all the password inputs (`/login`, `/signup` and `forgotpassword/reset/:id`). 👍